Fix wormhole opening events and mission

1010todd opened the pull request:
*  Fix wormhole opening event overriding entire system (which it probably shouldn't)
* Also fix broken wormhole because destination wormhole got overridden. (Might be intentional? If so it's a poor design.)
* Made wormhole opening mission offer on landing instead of spaceport.

kestrel1110 commented:
Updating scappers to make a little easier to capture, but requires at least 600 crew for success.

kestrel1110 commented:
Working on the thumbnails right now, they will be done soon.

1010todd committed:
Fix wormhole event overriding entire system

-Fix wormhole opening event overriding entire system (which it probably shouldn't)
-Also fix broken wormhole because destination wormhole got overridden. (Might be intentional? If so it's a poor design.)

1010todd committed:
Wormhole open on landing instead of at spaceport

Event triggering missions typically shouldn't require player to visit spaceport to trigger. I don't see sign of this having a reason to require visiting spaceport at the moment.

kestrel1110 commented:
cool
